hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -372,14 +372,15 @@ public StorageContainerManager(OzoneConfiguration conf,
   private void initializeSystemManagers(OzoneConfiguration conf,
                                        SCMConfigurator configurator)
       throws IOException {
+    clusterMap = new NetworkTopologyImpl(conf);
```

Comment:
We don't need to create two NetworkTopologyImpl instances. One here and one in SCMNodeManager. 
Just keep one is good.

Reply:
In SCMNodeManager constructor, it get the network topology instance from passed in scmManager parameter. Here is the line of code, 
L122:  this.clusterMap = scmManager.getClusterMap();